inc/saitunnel.h

```diff
@@ -822,17 +828,18 @@ typedef enum _sai_tunnel_term_table_entry_attr_t
     /**
      * @brief Tunnel termination IP address
      *
-     * @type sai_ip_address_t
+     * @type sai_ip_prefix_t
```

Comment:
this change is not backward compatible.

Reply:
Thanks for the comment. You are right I have changed the data type. However, you can still specify an IP address by providing a mask of all 1s, making it backwards compatible wrt the intent. In a nutshell, I am proposing to make the data type more generic to take advantage ASICs which support tunnel termination of prefixes.

Reply:
how, this still need all existing application to change their codes. can you introduce additional attribute to avoid the need to change existing applications?

Reply:
Thanks for the comment. I got the same feedback in today's SAI WG meeting. The consensus we reached in the meeting was to add new attributes for specifying src and dst IP mask. PTAL.